iconCompiler/svgCleaners/index.js

```diff
@@ -51,11 +51,11 @@ const readIcon = (file) =>
 
 // optimize using SVGO and custom clean options
 
-const optimizeIcon = ({ file, data }, SVGOPlugins, customOptims) =>
+const optimizeIcon = ({ file, data }, svgoPlugins, customOptims) =>
```

Comment:
Is it possible to put the changes in this file in another PR please? In order to separate the commits.